Libplanet/Store/IStore.cs

```diff
@@ -85,6 +65,80 @@ void PutBlock<T>(Block<T> block, Address addressesMask)
             AddressStateMap states
         );
 
+        /// <summary>
+        /// Lookup a state reference, which is a <see cref="Block{T}.Hash"/>
+        /// that has the state of the <paramref name="address"/>.
+        /// </summary>
+        /// <param name="namespace">The namespace to lookup a state reference.
+        /// </param>
+        /// <param name="address">The <see cref="Address"/> to find lookup.
+        /// </param>
+        /// <param name="lookupFrom">The starting point to lookup a state
+        /// reference. Blocks after <paramref name="lookupFrom"/> are ignored.
```

Comment:
How about changing the description like: <q>The upper bound (i.e., the latest block) of the search range.  Blocks after `lookupUntil` are ignored.</q>

Reply:
Changed description and parameter name.